index.html

```diff
@@ -403,6 +402,13 @@ <h3 id="bthome-data-format">BTHome Data format</h3>
             with a factor 0.01, this becomes a humidity of 50.55%.
           </li>
         </ul>
+        <p>
+          It is strongly advised to use object ids in numerical order (from low to high) in your advertisement.
```

Comment:
Instead of strongly advising it, why not make it mandatory?

Reply:
Agreed, changed it.